rep-0147.rst

```diff
@@ -187,15 +187,20 @@ The reference position is the origin of the ``base_link`` as defined in [#REP105
 Rate Interface
 ,,,,,,,,,,,,,,
 
-The messge to send a velocity command is ``geometry_msgs/Twist`` [#twist]_.
+The messge to send a velocity command is ``geometry_msgs/TwistStamped`` [#twiststamped]_.
 This is the standard way to command ROS vehicles.
 The datatype is not timestamped or externally referenced as it is not intended to be used in an open loop control system.
 
-* Datatype: ``geometry_msgs/Twist`` [#twist]_
-* Topic: ``cmd_vel``
+* Datatype: ``geometry_msgs/TwistStamped`` [#twiststamped]_
+* Topic: ``command_velocity``
 
 ::
 
+    std_msgs/Header header
+    uint32 seq
+    time stamp
+    string frame_id
+    geometry_msgs/Twist twist
```

Comment:
Rendering it this way is confusing. Indent these fields like ``rosmsg show`` would.